core/utils.js

```diff
@@ -131,21 +131,68 @@ Blockly.utils.getRelativeXY = function(element) {
     }
   }
 
-  // Third, check for style="transform: translate3d(...)".
+  // Then check for style = transform: translate(...) or translate3d(...) 
   var style = element.getAttribute('style');
-  if (style && style.indexOf('translate3d') > -1) {
-    var styleComponents = style.match(Blockly.utils.getRelativeXY.XY_3D_REGEX_);
+  if (style && style.indexOf('translate') > -1) {
+    var styleComponents = style.match(Blockly.utils.getRelativeXY.XY_2D_REGEX_);
+    // Try transform3d if 2d transform wasn't there.
+    if (!styleComponents) {
+      styleComponents = style.match(Blockly.utils.getRelativeXY.XY_3D_REGEX_);
+    }
     if (styleComponents) {
       xy.x += parseFloat(styleComponents[1]);
       if (styleComponents[3]) {
         xy.y += parseFloat(styleComponents[3]);
-      }
+       }
     }
   }
   return xy;
 };
 
 /**
+ * Return the coordinates of the top-left corner of this element relative to
+ * the div blockly was injected into. 
+ * @param {!Element} element SVG element to find the coordinates of.
```

Comment:
Add: "If this is not a child of the div blockly was injected into, the behaviour is undefined."